packages/abi-gen-wrappers/src/generated-wrappers/asset_proxy_owner.ts

```diff
@@ -105,15 +105,7 @@ export class AssetProxyOwnerContract extends BaseContract {
         ): Promise<string
         > {
             const self = this as any as AssetProxyOwnerContract;
-            const functionSignature = 'owners(uint256)';
-            const inputAbi = self._lookupAbi(functionSignature).inputs;
-            [index_0
-        ] = BaseContract._formatABIDataItemList(inputAbi, [index_0
-        ], BaseContract._bigNumberToString.bind(self));
-            BaseContract.strictArgumentEncodingCheck(inputAbi, [index_0
-        ]);
-            const ethersFunction = self._lookupEthersInterface(functionSignature).functions.owners;
-            const encodedData = ethersFunction.encode([index_0
+            const encodedData = self._strictEncodeArguments('owners(uint256)', [index_0
```

Comment:
Doesn't particularly matter since these are auto-generated, but I'm curious why the formatting is so strange (looks like it was this way before too).

Reply:
That is a good question. I always figured it had something to do with Handlebars. @LogvinovLeon any ideas?